proto/viam/app/packages/v1/packages.proto

```diff
@@ -102,6 +105,7 @@ message GetPackageRequest {
   string organization_id = 1;
   string name = 2;
   string version = 3;
+  string id = 4;
```

Comment:
Docs for this. And what the desired logic is for when org and name are supplied with id?  ^ same on delete.

Reply:
Ah sorry completely missed that and went right to the Files tab :(

Reply:
Note that just added `version` in the CreatePackageResponse to make it return everything you need to call Get/Delete.

Reply:
Should we also add the filed to sepcify if we should generated a signed URL? Can be in a follow up.